published/009-2016-March-28.md

```diff
@@ -0,0 +1,239 @@
+
+On a similar note, [gx](//github.com/whyrusleeping/gx), a package manager for Go, was also mentioned in a lot of the discussions about npm and package managers, especially on [Hacker News](https://news.ycombinator.com/). In the past two weeks, the project has recieved over 1000 stars - before, there were around 50.
+
+The discussion about how to best use IPFS as a package manager is ongoing. Jump on GitHub if you have something to say; we're listening in the [FAQ](https://github.com/ipfs/faq/issues) and in [the notes repo](https://github.com/ipfs/notes/issues).
+
+### DNS outage
+
+We're using DigitalOcean for providing ipfs.io DNS. On Tuesday, March 24th, DigitalOcean DNS was hit by a [severe outage](https://www.digitalocean.com/company/blog/update-on-the-march-24-2016-dns-outage/) lasting hours, which took the public gateway at ipfs.io down. We switched to [DNSimple](https://dnsimple.com) in an ad-hoc fashion and brought ipfs.io back while DigitalOcean was still down, but this incident obviously hit us on the wrong foot a bit. We'll be working to never get taken down this way again. Here's a few things we'll do:
+
+- codified DNS zones, and tools to upload them to DNS providers
+- one or two backup DNS providers
+- monitoring and failover procedures
+
+We'll post a more detailed postmorten on our blog in the next few days.
+
+### Captain.log
+
+#### Aye, you might want to check the new js-ipfs Captain.log entry, mate!
+
+Following [js-ipfs roadmap](https://github.com/ipfs/js-ipfs/blob/master/ROADMAP.md#milestone-1---js-ipfs-on-the-browser), we’re close™ to having a workable js-ipfs version that can be used in the browser and in Node.js. This will mark a very important milestone on the IPFS project and enable a whole set of new distributed web applications to be possible. If you want to be part of this effort, check out our [Captain.log](https://github.com/ipfs/js-ipfs/issues/30#issuecomment-205884015) entry to get a full update and a list of tasks you can contribute to.
+
+### Orbit
+
+@haadcode has been working on improvements to [orbit-db](https://github.com/haadcode/orbit-db), [ipfs-log](https://github.com/haadcode/ipfs-log/) and [Orbit](https://github.com/haadcode/orbit). The message history fetching is now more stable and the UI feedback for loading messages is fixed. All this work will improve the user experience of Orbit.
+
+### js-ipfs-init
+
+`js-ipfs init` works! @noffle finished the remaining pieces this week, including CLI usage. This included a handful of [auxiliary](https://github.com/ipfs/js-ipfs/issue/91) [PRs](https://github.com/ipfs/js-ipfs/issue/96) that cascaded out of that work. This makes the `js-ipfs` init process produce IPFS repos that are compatible with go-ipfs' - this is big!
```

Comment:
use qualifiers like "this is big" carefully. i dont see why this is a big deal worth calling out over other things in this weekly